README.md

```diff
@@ -1,330 +1,307 @@
-# ROBLOX IS KILLING LOCAL CLIENTSETTINGS LOADING
-# NO MORE CLOSET CHEATERS
-![drake](https://github.com/pizzaboxer/bloxstrap/assets/67791946/303dfde5-1968-435e-8660-249e8b6001c6)
+> [!NOTE]
+> Local Fast Flags are being patched out. Expect your flags to not work soon enough.
 
-# Graphical Settings
+![drake](https://github.com/pizzaboxer/bloxstrap/assets/67791946/303dfde5-1968-435e-8660-249e8b6001c6) 
 
-### Disable PostFX
-[Link To Original Post](https://discord.com/channels/1099468797410283540/1150655466830041128)
-```json
-{
-    "FFlagDisablePostFx": true
-}
-```
+# Change Rendering Mode
 
-### Change Rendering Mode
```

Comment:
Keep the "Change Rendering Mode" section same as the current one in the official repo.

Reply:
I already did this.